rc/tools/lint.kak

```diff
@@ -46,9 +49,8 @@ define-command \
         dir=$(mktemp -d "${TMPDIR:-/tmp}"/kak-lint.XXXXXXXX)
 
         # A fifo to send the results back to a Kakoune buffer.
-        # FIXME: Should we put the lint output in toolsclient?
         mkfifo "$dir"/fifo
-        printf '%s\n' "evaluate-commands -draft %{
+        printf '%s\n' "evaluate-commands -try-client '$kak_opt_toolsclient' %{
```

Comment:
I think the `-draft` should stay. The primary way of reading errors is via flag lines set on the buffer itself, I don't want the focus to change to another buffer every time I call `:lint`.

Reply:
I wasn't sure if it is possible to use both `-draft` and `-try-client`, since both are specifying a client context. Also, I don't get a buffer change when I use it without a `toolsclient` definition (which is a bit strange when I think about it), but let me know if it is happening for you.

Reply:
It seems that you are correct, using `-try-client` doesn't change the focus. I always thought it did. Sorry for the noise!